extensions/import/src/wizard/pages/modifyColumnsPage.ts

```diff
@@ -49,10 +50,12 @@ export class ModifyColumnsPage extends ImportPage {
 		{ name: 'varchar(50)', displayName: 'varchar(50)' },
 		{ name: 'varchar(MAX)', displayName: 'varchar(MAX)' }
 	];
+
 	private table: azdata.DeclarativeTableComponent;
 	private loading: azdata.LoadingComponent;
 	private text: azdata.TextComponent;
 	private form: azdata.FormContainer;
+	private checkAllNullabales: azdata.CheckBoxComponent;
```

Comment:
If we're going to be doing this might as well add one for Primary Key as well IMO.

Although really - this is a general "nice to have feature" for any table so I'd like to see this be a core part of the declarative table and not have to redo it every time we want this functionality.